ant-chapters/chapter1.tex

```diff
@@ -1,5 +1,103 @@
 \chapter{Rings of integers}\llabel{ring-of-integers}
-When we have a field extension $L$ of $\Q$, we would like to define a ring of integers for $L$, with properties similar to the ring $\Z\subeq \Q$. We will define this ring of integers in a slightly more general context.
+
+We begin with a hint of applications of rings of integers for motivation. We then introduce algebraic numbers to define a few key terms illustrating these concepts with concrete examples. Following that, we will generalize our definitions to arbitrary fields.
+
+\section{Motivation}
+A natural way to begin a book about Algebraic Number Theory is by introducing number fields. While we will define them much more precisely and generally soon,
+we may temporarily think of them as fields of rational numbers extended by ``external" elements. Rings of integers are similar, but are based on $\Z$ instead.
+
+\begin{ex} A typical ring of integers is $\Z[\sqrt{2}] = \left\{m + n \sqrt{2} : m,n \in \Z\right\}$.
+\end{ex}
+
+\noindent This construction is very powerful, and can be used to solve many classical problems, such as:
+\begin{itemize}
+\item \textbf{Diophantine Equations}: For example, solving the Diophantine equation $x^2 - dy^2 = 1$ (Pell's equation) can be understood as a problem in $\Z(\sqrt{d})$. Without going into detail for now, norm of $x + y\sqrt{d}$ in such ring is defined as $x^2 - dy^2$, which corresponds to equation we're interested in.

```

Comment:
Should I nitpick that if d ≡ 1 (mod 4) we're actually working in `Z[1/2(1 + sqrt(d)]`? This doesn't sound too important in the motivational segment.

Reply:
It's not clear what Z(.) means. I'd stick with writing Z[.]. You can add a footnote noting that we consider Z[1/2(1 + √d)] when d ≡ 1 (mod 4).

Reply:
That's actually me being sloppy and not proofreading enough. I wasn't trying to use a clever notation. Thanks for spotting (I've removed this part entirely for now, though).